CMP1903_A1_2324/Die.cs

```diff
@@ -12,11 +13,36 @@ internal class Die
          * The Die class should contain one property to hold the current die value,
          * and one method that rolls the die, returns and integer and takes no parameters.
          */
+        private int _DieValue;
```

Comment:
Refer to C# style guidelines to name variable appropriately. _camelCase; therefore it should be _dieValue;